### PR TITLE
Fix typo case in the phpunit command.

### PR DESCRIPTION
**Pull request by pierregaste**
Another typo with the case in the commande line :)

### CODE REVIEW

File: knpu/episode5/hal-json.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Add `.href` to this to match the new format:
 Try it out!
 
 ```bash
-vendor/bin/phpunit --filter testGetProgrammer
```
Comment (bocharsky-bw) [inline knpu/episode5/hal-json.md:50]: Could you also tweak the same method name on line 39?

**Comment by bocharsky-bw** [comment]
And in one more place, please:
- episode3/conditionally-embedding-relations.md (testGETProgrammer)
- episode5/customize-link-format.md (testGETProgrammerDeep)

**Comment by pierregaste** [comment]
@bocharsky-bw done!

I just finished all the videos, I should no longer bother you ^^

**Comment by bocharsky-bw** [comment]
Kudos!

Haha, don't worry about bothering me. You help to improve our screencasts - that's awesome ;)

Thanks for all your fixes so far!

UPD: Deployed!

**Pull request merged**